Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -24,20 +20,17 @@ final class PledgeDescriptionCell: UITableViewCell, ValueCell {
 
   // MARK: - Properties
 
-  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
-  private lazy var containerImageView: UIView = {
-    UIView(frame: .zero) |> \.translatesAutoresizingMaskIntoConstraints .~ false
-  }()
-
-  private lazy var pledgeImageView: UIImageView = {
-    UIImageView(frame: .zero) |> \.translatesAutoresizingMaskIntoConstraints .~ false
-  }()
-
+  private lazy var dateLabel: UILabel = { UILabel(frame: .zero) }()
   private lazy var descriptionStackView: UIStackView = { UIStackView(frame: .zero) }()
   private lazy var estimatedDeliveryLabel: UILabel = { UILabel(frame: .zero) }()
-  private lazy var dateLabel: UILabel = { UILabel(frame: .zero) }()
-  private lazy var spacerView = UIView(frame: .zero)
   private lazy var learnMoreTextView: UITextView = { UITextView(frame: .zero) |> \.delegate .~ self }()
+  private lazy var rewardCardContainerMaskView: UIView = { UIView(frame: .zero) }()
+  private var rewardCardContainerMaskViewHeightConstraint: NSLayoutConstraint?
+  private var rewardCardContainerMaskViewWidthConstraint: NSLayoutConstraint?
+  private lazy var rewardCardContainerView: UIView = { UIView(frame: .zero) }()
+  private lazy var rewardCardView: RewardCardView = { RewardCardView(frame: .zero) |> \.delegate .~ self }()
```

Comment:
`rewardCardContainerMaskView` is the view that we add to our cell's `stackView` and set explicit `width` and `height` constraints on. This is used to mask the bottom of the thumbnail because although it's being scaled down it can still be indeterminately long.

`rewardCardContainerView` contains the `rewardCardView` but adds necessary `layoutMargins`.

`rewardCardView` is the card itself without the pledge button.